taworks/taform/templates/taform/application.html

```diff
@@ -172,7 +173,7 @@ <h2>Course Application</h2>
     </div>
 {% for i in courses %}
 <div class="required">
-    <div id="row">
+    <div class="row">
         <div id="left">
```

Comment:
can you change line 177 and line 178 to classes as well? they should have never been ids.